codegen/lib/graphql_java_gen/templates/Enum.java.erb

```diff
@@ -0,0 +1,69 @@
+// Generated from <%= script_name %>
+
+package <%= package_name %>;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.shopify.graphql.support.AbstractResponse;
+import com.shopify.graphql.support.Arguments;
+import com.shopify.graphql.support.Error;
+import com.shopify.graphql.support.Query;
+import com.shopify.graphql.support.SchemaViolationError;
+import com.shopify.graphql.support.TopLevelResponse;
+import com.shopify.graphql.support.Input;
+<% imports.each do |import| %>
+  import <%= import %>;
+<% end %>
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+    <% case type.kind when 'OBJECT', 'INTERFACE', 'UNION' %>
```

Comment:
Is this needed? We're in `Enum.java.erb` it should only ever be called when the kind is `ENUM`?

Reply:
No, I probably just missed this line when deleting stuff